core-feature-pack/src/main/resources/content/bin/standalone.sh

```diff
@@ -262,18 +262,28 @@ if [ "$PRESERVE_JAVA_OPTS" != "true" ]; then
         fi
     fi
 
+    # Set flag if JVM is modular
+    setModularJvm
+
     if [ "$GC_LOG" = "true" ]; then
         # Enable rotating GC logs if the JVM supports it and GC logs are not already enabled
-        NO_GC_LOG_ROTATE=`echo $JAVA_OPTS | $GREP "\-verbose:gc"`
+        mkdir -p $JBOSS_LOG_DIR
+        NO_GC_LOG_ROTATE=`echo $JAVA_OPTS | $GREP "\-Xlog\:\?gc"`
         if [ "x$NO_GC_LOG_ROTATE" = "x" ]; then
             # backup prior gc logs
+            mv -f "$JBOSS_LOG_DIR/gc.log" "$JBOSS_LOG_DIR/backupgc.log" >/dev/null 2>&1
             mv -f "$JBOSS_LOG_DIR/gc.log.0" "$JBOSS_LOG_DIR/backupgc.log.0" >/dev/null 2>&1
             mv -f "$JBOSS_LOG_DIR/gc.log.1" "$JBOSS_LOG_DIR/backupgc.log.1" >/dev/null 2>&1
             mv -f "$JBOSS_LOG_DIR/gc.log.2" "$JBOSS_LOG_DIR/backupgc.log.2" >/dev/null 2>&1
             mv -f "$JBOSS_LOG_DIR/gc.log.3" "$JBOSS_LOG_DIR/backupgc.log.3" >/dev/null 2>&1
-            mv -f "$JBOSS_LOG_DIR/gc.log.4" "$JBOSS_LOG_DIR/backupgc.log.4" >/dev/null 2>&1
```

Comment:
Why we need remove this line? I start WF with JDK8, invoke some rest end-point that consumes a lot of memory and:
```
[mkopecky@dhcp-10-40-4-197 wildfly]$ ll standalone/log/
total 4360
-rw-rw-r--. 1 mkopecky mkopecky       0 Sep 20 09:44 audit.log
-rw-rw-r--. 1 mkopecky mkopecky 1048832 Sep 20 09:54 gc.log.0
-rw-rw-r--. 1 mkopecky mkopecky 1048826 Sep 20 09:56 gc.log.1
-rw-rw-r--. 1 mkopecky mkopecky 1048821 Sep 20 09:58 gc.log.2
-rw-rw-r--. 1 mkopecky mkopecky  235106 Sep 20 09:58 gc.log.3.current
-rw-rw-r--. 1 mkopecky mkopecky 1048832 Sep 20 09:53 gc.log.4
-rw-rw-r--. 1 mkopecky mkopecky   13813 Sep 20 09:44 server.log
[mkopecky@dhcp-10-40-4-197 wildfly]$
```

Reply:
@marekkopecky copy-paste issue. Line is back